rclcpp/include/rclcpp/node_options.hpp

```diff
@@ -152,6 +152,22 @@ class NodeOptions
   NodeOptions &
   use_global_arguments(bool use_global_arguments);
 
+  /// Return the enable_rosout flag.
+  RCLCPP_PUBLIC
+  bool
+  enable_rosout() const;
+
+  /// Set the enable_rosout flag, return this for parameter idiom.
+  /**
+   * If false this will cause the node not to use rosout logging
+   *
+   * Defaults to true for now, as there are still some cases where it is not
```

Comment:
@Barry-Xu-2018 this statement is a bit vague, and it reads funny. Did you mean to say _"... where it IS desirable."_?

Reply:
Yes.  It is my mistake.